velox/exec/RowContainer.h

```diff
@@ -983,8 +983,8 @@ class RowContainer {
       return compareComplexType(row, offset, decoded, index) == 0;
     }
     if (Kind == TypeKind::VARCHAR || Kind == TypeKind::VARBINARY) {
-      return compareStringAsc(
-                 valueAt<StringView>(row, offset), decoded, index) == 0;
+      return compareStringEqual(
```

Comment:
Looks like this is the problematic change that breaks aggregations and was just reverted. Let's make sure it doesn't sneak back in accidentally. CC: @xiaoxmeng